content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info-clio.md

```diff
@@ -253,47 +676,52 @@ The `info` object may have some arrangement of the following fields:
+| `rippled_version`                   | String          |  The version number of the running `rippled` server that the `Clio` server is connected to. |
 | `validated_ledger`                  | Object          | _(May be omitted)_ Information about the most recent fully-validated ledger. If the most recent validated ledger is not available, the response omits this field and includes `closed_ledger` instead. |
 | `validated_ledger.age`              | Number          | The time since the ledger was closed, in seconds. |
 | `validated_ledger.base_fee_xrp`     | Number          | Base fee, in XRP. This may be represented in scientific notation such as `1e-05` for 0.00001. |
 | `validated_ledger.hash`             | String          | Unique hash for the ledger, as hexadecimal. |
 | `validated_ledger.reserve_base_xrp` | Number          | Minimum amount of XRP (not drops) necessary for every account to keep in reserve |
 | `validated_ledger.reserve_inc_xrp`  | Number          | Amount of XRP (not drops) added to the account reserve for each object an account owns in the ledger. |
 | `validated_ledger.seq`              | Number          | The [ledger index][] of the latest validated ledger. |
-| `validation_quorum`                 | Number          | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
 | `validator_list_expires`            | String          | _(Admin only)_ Either the human readable time, in UTC, when the current validator list will expire, the string `unknown` if the server has yet to load a published validator list or the string `never` if the server uses a static validator list. [Updated in: rippled 1.5.0][] |
+| `cache`                             | Object          |  Data retrieved from the `rippled` server is stored in efficiently in the `cache` object. Thus resulting in a higher throughput and sometimes lower latency for Clio API requests. |
+| `cache.size`                        | Number          |  Size of the `cache` object containing the validated ledger data retrieved from P2P `rippled` server. |
+| `cache.is_full`                     | Boolean         | This is true if the cache is full or not empty. |
+| `cache.latest_ledger_seq`           | Number          | The [ledger index][] of the latest validated ledger stored in the cache. |
+| `etl`                               | Object          | The `rippled` sources (ETL sources) that the Clio server is connected to. |
+| `etl.etl_sources`                   | Object Array    | List the `rippled` sources (ETL sources) that the Clio server is connected to and extracts data from. |
+| `etl.etl_sources.validated_range`   | String          | The validated ledger range retrieved by the P2P `rippled` server. |
+| `etl.etl_sources.is_connected`      | Boolean         |  |
```

Comment:
True if clio is connected to this source via websocket, false otherwise. A value of false here could indicate a networking issue, or that rippled is not running, amongst other things.